src/erlang_ls.erl

```diff
@@ -111,7 +111,6 @@ set(transport, Name) ->
 set(port, Port) ->
   application:set_env(?APP, port, Port);
 set(log_dir, Dir) ->
-  application:set_env(?APP, logging_enabled, true),
   application:set_env(?APP, log_dir, Dir);
 set(log_level, Level) ->
   application:set_env(?APP, log_level, Level);
```

Comment:
I just noticed that the rewrite of parsing the options using `getopt` removed the enabling of logging when a `log_level` is provided. I guess it's saner to just enable it in one place or have it enabled by default.

Reply:
Sure, feel free to raise an issue.